sdk/nodejs/output.ts

```diff
@@ -577,6 +583,9 @@ export function unsecret<T>(val: Output<T>): Output<T> {
     );
 }
 
+/**
+ * [isSecret] returns `true` if and only if the provided [Output] is a secret.
```

Comment:
Why are `isSecret` and `Output` in brackets here?

Reply:
It's a hack TBH -- it's an incomplete Markdown link (`[]()`), so e.g. if you look at hover docs, the identifier will be highlighted neatly. The right way to do this is with `{@link isSecret}` (for instance, though you'd likely not do it here when documenting that function itself). Here I've done it to match the functions around it, though the "correct" thing to do is wholesale rewrite all our SDK docs to use linking correctly and more thoroughly.